homeassistant/helpers/schema_config_entry_flow.py

```diff
@@ -101,6 +101,11 @@ def parent_handler(self) -> SchemaConfigFlowHandler | SchemaOptionsFlowHandler:
         """Return parent handler."""
         return self._handler
 
+    @property
+    def options(self) -> dict[str, Any]:
```

Comment:
Is this included in another PR already?

Reply:
The other PR added the property to `OptionsFlowWithConfigEntry`
In this case, we want to expose the property on `SchemaCommonFlowHandler` so we don't have to assert if it's `SchemaOptionsFlowHandler` (versus `SchemaConfigFlowHandler`)